pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/service/WebSocketProxyConfiguration.java

```diff
@@ -32,6 +32,8 @@
     public static final int GLOBAL_ZK_THREADS = 8;
     // Number of IO threads in Pulsar Client
     public static final int PULSAR_CLIENT_IO_THREADS = Runtime.getRuntime().availableProcessors();
+    // Number of connections per Broker in Pulsar Client
+    public static final int PULSAR_CLIENT_CONNECTIONS_PER_BROKER = Runtime.getRuntime().availableProcessors();
```

Comment:
We should make it configurable and initialize it to available processors if not set in the config.

Do you have the performance numbers?